app/controllers/api/v1/packages_controller.rb

```diff
@@ -212,7 +212,12 @@ def package_valuation
       end
 
       def print_inventory_label
-        PrintLabelJob.perform_later(@package.id, User.current_user.id, "inventory_label", print_count)
+        opts = {
```

Comment:
It's probably better to lookup the printer at this step. That way if the printer isn't set for this tag, you can return an error.

Reply:
ohh.. thanks for pointing it out, will add check for this.